src/addressfinder/reverseGeocode.tsx

```diff
@@ -0,0 +1,41 @@
+const addressFinderAPIKey = "ARFHPVK67QXM49BEWDL3";
+const reverseGeocodeURL = `https://api.addressfinder.io/api/nz/address/reverse_geocode/?key=${addressFinderAPIKey}&format=json`;
+
+export interface reverseGeocodeResp {
+  completions: addressResp[];
+  paid: boolean;
+  demo: boolean;
+  success: boolean;
+}
+
+interface addressResp {
+  a: string;
+  pxid: string;
+}
```

Comment:
Please capitalise interface to `AddressResponse` and add a comment on line 12 and 13 so that devs know what `a` and `pxid ` mean 🙏

Reply:
Done.